TitanRobotics2024/src/main/java/frc/robot/Auto/Actions/TurnDegreesAction.java

```diff
@@ -0,0 +1,97 @@
+package frc.robot.Auto.Actions;
+
+import frc.robot.Subsystem.DriveBase;
+import frc.robot.Subsystem.Gyro;
+import frc.robot.Subsystem.PositionEstimation;
+
+import javax.swing.text.Position;
+
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.SPI;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+
+public class TurnDegreesAction implements Actions 
+{
+    Timer timer;
+    private double currentDegrees = 0;
+    private double desiredDegrees;
+    private double targetDegrees;
+    private double turn;
+    private double toleranceDegrees = 0.0025;
+    private double endAfterSeconds;
+
+    private DriveBase driveBase;
+    private Gyro gyro;
+    private PositionEstimation position;
+
+    private PIDController PID;
+    private double kp = 0.01;
```

Comment:
You are re-using these variables (TitanRobotics2024/src/main/java/frc/robot/Auto/Actions/DriveForDistanceAction.java), move them to a constants file

Reply:
they may very well change to not be shared numbers, they are just copied over